code/game/gamemodes/gang/recaller.dm

```diff
@@ -280,10 +280,13 @@
 						usr << "<span class='warning'>There's not enough room here!</span>"
 						return
 
-				if(gang.points >= 30)
-					item_type = /obj/machinery/dominator
-					usr << "<span class='notice'>The <b>dominator</b> will secure your gang's dominance over the station. Turn it on when you are ready to defend it.</span>"
-					pointcost = 30
+				if(usrarea.type in gang.territory | gang.territory_new)
```

Comment:
@Razharas we don't use spaces for the binary version of ```|```, so we don't use spaces for the list form either, this is almost CERTAINLY what confused @duncathan 

```||``` has spaces, ```|``` does not, the lack of spaces with ```|``` help show it's a merged list.